pkgs/by-name/sw/swhkd/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+let
+  name = "swhkd";
+  version = "30f25b5bf99df5f16d91b51a7bd397c1de075085";
+  buildInputs = with pkgs; [ systemd ];
```

Comment:
From what I glanced at upstream page, `systemd` should not be hard dependency.

Consider moving systemd-related stuff into separate output and adding `withSystemd` flag for the benefit of platforms that do not have systemd (e.g static platform).

Reply:
Hey,

[This answer](https://discourse.nixos.org/t/solved-libudev-replaced-by-udev/18951/3?u=ivandimitrov8080) showed me how to fix a build failure. If I don't specify systemd it doesn't build.

Reply:
Ah, I see. Can you then please add `meta.broken = !(lib.meta.availableOn stdenv.hostPlatform.systemd)`. Will make error cleaner for somebody who will try to build on unsupported system.